_config.yml

```diff
@@ -12,17 +12,17 @@ description: >
   Student Robotics is an exciting annual competition challenging people aged
   16-19 to assemble a team and build fully autonomous robots.
 
-year: 2024
+year: 2025
 api_root: https://studentrobotics.org/comp-api
 
-challenges_link: https://studentrobotics.org/docs/resources/2024/rulebook.html#challenges
-virtual_competition_link: https://studentrobotics.org/events/sr2024/virtual-competition/
-competition_link: https://studentrobotics.org/events/sr2024/competition/
-youtube_stream_link: https://www.youtube-nocookie.com/embed/CpCC2fTn0os
-venue_layout_link: https://studentrobotics.org/resources/sr2024/venue-map.pdf
+challenges_link: https://studentrobotics.org/docs/resources/2025/rulebook.html#challenges
+virtual_competition_link: https://studentrobotics.org/events/sr2025/virtual-competition/
+# competition_link: https://studentrobotics.org/events/sr2025/competition/
```

Comment:
Thought: Is it worth setting this to the same as `virtual_competition_link`, so the "Event Information" link at the top of the page goes to the virtual comp page? Then when we get to the physical competition, we can replace it.

Reply:
Yeah that sounds like a good shout